FESS-GUI/mainwindow.cpp

```diff
@@ -522,19 +518,157 @@ void MainWindow::on_lineEditPassword_textEdited(const QString &password)
         ui->pushButton_ApplySettings->setEnabled(true);
 }
 
-
-
 void MainWindow::on_actionLock_frame_rate_at_30FPS_triggered(bool checked)
 {
-    dataTimer->stop();
     if(checked)
-        refreshRate = 33;
+    {
+        graphRefreshRate = 30;
+        graphRefreshTimer->setInterval(refreshRateToMS(graphRefreshRate));
+    }
     else
-        refreshRate = 10;
-    dataTimer->start(refreshRate);
+    {
+        graphRefreshRate = 100;
+        graphRefreshTimer->setInterval(refreshRateToMS(graphRefreshRate));
+    }
 }
 
 void MainWindow::on_actionLock_graph_scale_to_max_value_triggered(bool checked)
 {
 
 }
+
+// Information Messages
+
+
+// Error Messages
+
+void MainWindow::errorInterafceNotDefined()
+{
+    ui->errorLog->append(QString("%1 : %2").arg(QTime::currentTime().toString(),ERROR_INTERFACE_NOT_SET));
+    errorHandler->showMessage(ERROR_INTERFACE_NOT_SET);
+}
+
+
+// Flywheel Operations
+void MainWindow::runFlywheelOperations()
+{
+    flywheelOperation->sync();
+}
+
+void MainWindow::startFlywheelInterface()
+{
+    deviceInterface = interfaceManager->getCurrentInterface();
+
+    if(!deviceInterface)
```

Comment:
Have to disagree on this one 

"In C and C++ programming, two null pointers are guaranteed to compare equal; ANSI C guarantees that any null pointer will be equal to 0 in a comparison with an integer type; furthermore the macro NULL is defined as a null pointer constant, that is value 0 (either as an integer type or converted to a pointer to void), so a null pointer will compare equal to NULL."